CMakeLists.txt

```diff
@@ -79,8 +79,8 @@ else()
 endif()
 
 macro(nbl_adjust_flags)
-	option(NBL_GCC_SANITIZE_ADDRESS OFF)
-	option(NBL_GCC_SANITIZE_THREAD OFF)
+	option(NBL_SANITIZE_ADDRESS OFF)
+	option(NBL_GCC_SANITIZE_THREAD OFF) # @Anastazluk did MSVC finally include the thread sanitizer?
```

Comment:
we probably shouldn't mention this is even an option if the compiler is MSVC (only GCC and Clang have a THREAD sanitizer)

Reply:
rename to `NBL_SANITIZE_THREAD` and only show when compiler is GCC or CLANG

Reply:
assign @AnastaZIuk

Reply:
@AnastaZIuk did it get done?

Reply:
done in following commit of track of `input_channel_demo` branch